website/integrations/services/engomo/index.md

```diff
@@ -0,0 +1,89 @@
+
+-   `engomo.company` is the FQDN of the Engomo install.
+-   `authentik.company` is the FQDN of the authentik install.
+-   `engomo.mapping` is the name of the Scope Mapping.
+-   `ak.cert` = The authentik self-signed certificate you use for the service provider.
+
+## authentik configuration
+
+In authentik, create a new scope mapping. To do so, log in to the Admin interface, and navigate to **Customization-->Property Mapping** and then click **Create**.
+
+-   `engomo.mapping` is the value for the Name.
+-   `profile` is the value for the Scope name.
+-   `return {"preferred_username": request.user.email,}` is the value for the Expression.
+
+Create an application and OAuth2/OpenID provider in authentik. Use the following parameters for the OAuth2/OpenID provider::
+
+Provider:
+
+-   Name: `SP-engomo`
+-   Client type: `Public`    
+-   Redirect URIs/Origins (RegEx): `https://engomo.company/auth` and `com.engomo.engomo://callback/`
+-   Signing Key: `ak.cert`
+-   Scopes: `authentik default OAuth Mapping: OpenID 'email'` and `authentik default OAuth Mapping: OpenID 'offline_access'` and `authentik default OAuth Mapping: OpenID 'openid'` and `engomo.mapping`
+-   Note the Client ID and Client Secret.
+
+> [!IMPORTANT]
+> Redirect URIs => write the values line by line.
+
+Leave the rest as default values. You can of course adjust durations.
+
+Application:
+
+-   Name: `Engomo`
+-   Slug: `engomo`
+-   Launch URL: `https://engomo.company/`
+
+## engomo configuration
+
+Navigate to `https://engomo.company/composer` and log in with your admin credentials.
+
+-   Select `Server`.
+-   Select `Authentication`.
+-   Add a new authentication method by clicking on the plus icon on the right.
+-   Name: `authentik`
+-   Type: `OpenID Connect`
+-   Click **Create**.
+-   Set the `Issuer` to the authentik FQDN `https://authentik.company/application/o/engomo`.
+-   Set the `Client ID` to the Client ID from the SP-engomo provider that you created in authentik.
+-   Set the `Client Secret` to the Client Secret from the SP-engomo provider that you created in authentik.
+
+Leave the rest as default.
+
+## engomo user creation
+
+engomo doesn't create users automatically on signing in. So you have to do it manually right now.
+Navigate to `https://engomo.company/composer` and log in with your admin credentials.
+
+- Select `Users & Devices`.
+- Click the plus button next in the Users section.
+- Select `authentik` as the Authenticator in the dropdown.
+- Create your user by typing in the email as the Username used in authentik.
+
+At this point you are done.
+
+## testing the login
+
+- Open a browser of your choice and open the URL `https://engomo.company`.
+- Enter the created user's email adress and hit the small arrow icon to log in.
+- You should be redirected to authentik with all its login flows you have created by yourself.
+- If you are redirected back to the `https://engomo.company/composer` URL you did everything correct.
```

Comment:
Hmmmm... so is it that the user is first redirected back to authentik, and then on to https://engomo.company/composer?

If Yes, maybe clarify by saying something like:

"You should be redirected to authentik (via the with login flows you created) and then authentik should redirect you back to the `https://engomo.company/composer` URL."

Reply:
Also, thanks for adding these verification steps!!

Reply:
done